integration/setup/teardown.go

```diff
@@ -0,0 +1,29 @@
+// +build k8srequired
+
+package setup
+
+import (
+	"context"
+
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/microerror"
+)
+
+func teardown(ctx context.Context, config Config) error {
+	// clean host cluster components
+	err := framework.HelmCmd("delete --purge giantswarm-app-operator")
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	// clean guest cluster components
```

Comment:
```suggestion
	// clean tenant cluster components
```

Is this needed? I'm not sure this chart is actually installed.